pkg/domain/infra/abi/play.go

```diff
@@ -114,6 +117,24 @@ func (ic *ContainerEngine) PlayKube(ctx context.Context, body io.Reader, options
 	report := &entities.PlayKubeReport{}
 	validKinds := 0
 
+	// create default kube network if it doesn't exists
```

Comment:
This network should only be created in the block below where we do if len(options.Networks) ==0